mscore/pianoroll/pianoview.cpp

```diff
@@ -564,16 +566,27 @@ void PianoView::wheelEvent(QWheelEvent* event)
 //   showPopupMenu
 //---------------------------------------------------------
 
-void PianoView::showPopupMenu(const QPoint& pos)
+void PianoView::showPopupMenu(const QPoint& posGlobal)
       {
       QMenu popup(this);
-//      popup.addAction(getAction("cut"));
-//      popup.addAction(getAction("copy"));
-      popup.addAction(getAction("paste"));
-//      popup.addAction(getAction("swap"));
+
+      QAction* act;
+
+      act = new QAction(tr("Cut notes"));
+      connect(act, SIGNAL(triggered(bool)), this, SLOT(cutNotes()));
```

Comment:
It’s better not to use the old way of connections, but to use a modern:
connect(act, &QAction::triggered, this, &PianoView::cutNotes);

Reply:
Rarely used in the current source though